src/light.ts

```diff
@@ -0,0 +1,169 @@
+  }
+
+  if (entity.id === "test_mediaplayer") {
+    console.log("Got %s media-player command request: %s", entity.id, cmdId, params || "");
+
+    return uc.StatusCodes.Ok;
+  }
+
+  console.log("Got %s command request: %s", entity.id, cmdId);
+
+  return uc.StatusCodes.Ok;
+};
+
+/**
+ * Dedicated light entity command handler.
+ *
+ * Called by the integration-API if a command is sent to a configured light-entity.
+ *
+ * @param {Entity} entity light entity
+ * @param {string} cmdId command
+ * @param {?Object<string, *>} params optional command parameters
+ * @return {Promise<string>} status of the command
+ */
+const lightCmdHandler: CommandHandler = async function (entity, cmdId, params): Promise<StatusCodes> {
+  console.log("Got %s command request: %s", entity.id, cmdId);
+
+  // in this example we just update the entity, but in reality, you'd turn on the light with your integration
+  // and handle the events separately for updating the configured entities
+  switch (cmdId) {
+    case uc.entities.Light.Commands.Toggle:
+      if (entity.attributes?.state === uc.entities.Light.States.Off) {
+        uc.getConfiguredEntities().updateEntityAttributes(entity.id, {
+          [uc.entities.Light.Attributes.State]: uc.entities.Light.States.On,
+          [uc.entities.Light.Attributes.Brightness]: 255
+        });
+      } else if (entity.attributes?.state === uc.entities.Light.States.On) {
+        uc.getConfiguredEntities().updateEntityAttributes(entity.id, {
+          [uc.entities.Light.Attributes.State]: uc.entities.Light.States.Off,
+          [uc.entities.Light.Attributes.Brightness]: 0
+        });
+      }
+      break;
+    case uc.entities.Light.Commands.On:
+      // params is optional! Use a default if not provided.
+      // A real lamp might store the last brightness value, otherwise the integration could also keep track of the last value.
+      uc.getConfiguredEntities().updateEntityAttributes(entity.id, {
+        [uc.entities.Light.Attributes.State]: uc.entities.Light.States.On,
+        [uc.entities.Light.Attributes.Brightness]: params && params.brightness ? params.brightness : 127
+      });
+      uc.getConfiguredEntities().updateEntityAttributes("test_mediaplayer", {
+        [uc.entities.MediaPlayer.Attributes.Volume]: 24
+      });
+      break;
+    case uc.entities.Light.Commands.Off:
+      uc.getConfiguredEntities().updateEntityAttributes(entity.id, {
+        [uc.entities.Light.Attributes.State]: uc.entities.Light.States.Off,
+        [uc.entities.Light.Attributes.Brightness]: 0
+      });
+      break;
+    default:
+      return uc.StatusCodes.NotImplemented;
+  }
+
+  return uc.StatusCodes.Ok;
+};
+
+// create a light entity
+// normally you'd create this where your driver exposed the available entities
+// The entity name can either be string (which will be mapped to english), or a Map with multiple language entries.
+const name = {
+  ["de"]: "Mein Lieblingslicht",
+  ["en"]: "My favorite light"
+};
```

Comment:
I think the language object `name` can be simplified:
```ts
const name = {
  de: "Mein Lieblingslicht",
  en: "My favorite light"
};
```
tsc doesn't complain and the example still works :-)

Reply:
I updated this part.